fdbserver/tester.actor.cpp

```diff
@@ -1754,17 +1776,18 @@ ACTOR Future<Void> runTests(Reference<IClusterConnectionRecord> connRecord,
 		ifs.close();
 	}
 
+	knobProtectiveGroup = std::make_unique<KnobProtectiveGroup>(testSet.overrideKnobs);
```

Comment:
Is `knobProtectiveGroup` used anywhere in the code?

Reply:
`knobProtectiveGroup` is internally functioning by constructor and destructor.